book_maker/translator/chatgptapi_translator.py

```diff
@@ -388,3 +407,224 @@ def set_model_list(self, model_list):
         model_list = list(set(model_list))
         print(f"Using model list {model_list}")
         self.model_list = cycle(model_list)
+
+    def batch_init(self, book_name):
+        self.book_name = self.sanitize_book_name(book_name)
```

Comment:
I think this name is not support windows 
can we use  `pathlib` or `os.path`

Reply:
I've fixed it
https://github.com/yihong0618/bilingual_book_maker/pull/423#discussion_r1722606436